bin/linter.dart

```diff
@@ -173,8 +173,9 @@ void runLinter(List<String> args, LinterOptions initialLintOptions) {
         fileRoot: commonRoot,
         showStatistics: stats,
         machineOutput: options['machine'],
-        quiet: options['quiet'])..write();
-  } catch (err, stack) {
+        quiet: options['quiet'])
+      ..write();
+  } on Exception catch (err, stack) {
```

Comment:
This is not semantically equivalent, and I think we want to catch everything here. Which means that I think we should ignore the lint here (assuming that we enable it at all).